dolfin_dg/operators.py

```diff
@@ -570,7 +570,7 @@ def F_v(U, grad_U):
 
             grad_rho = grad_U[0, :]
             grad_rhou = ufl.as_tensor([grad_U[j, :] for j in range(1, dim + 1)])
-            grad_rhoE = grad_U[-1, :]
+            grad_rhoE = grad_U[dim+1, :]
```

Comment:
@nate-sime: Using [-1,:] did not work with ufl anymore.  Could you double check this please.